modules/test/playwright/tests/document-library-web/config.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {devices} from '@playwright/test';
+
+export const config = {
+	expect: {
+		timeout: 30 * 1000,
+	},
```

Comment:
Do you need to increment the global expect timeout here? Maybe not

Reply:
Probably not, I just copied it from other project... lets remove it ;-)